clonecatalog.py

```diff
@@ -323,7 +329,7 @@ def _get_or_create_transfer(
             try:
                 if securable_type == catalog.SecurableType.TABLE:
                     spark.sql(
-                        f"CREATE TABLE {target_securable_full_name} DEEP CLONE {source_securable_full_name}"
+                        f"CREATE OR REPLACE TABLE {target_securable_full_name} DEEP CLONE {source_securable_full_name}"
```

Comment:
As I understand, this line forces the deep cloning, but I don't see that the variable force_deep_clone is being used! shouldn't be there an if-else logic?

Reply:
The way the code was done, this deep clone only happens if there is an exception. That exception is given or not by the .get() method in the sdk that checks if the table exists. So if you do .get and it's not there it will throw an exception and it will execute the deep clone.
If you see the other change you commented, I raise an exception if the force_deep_clone is true and the .get doesn't happen. (it was just another way to force the exception).
I don't particularly like the method of deep cloning inside an exception, instead of using if/elses but that's what was implemented originally and I din't want to change too much.